src/main/resources/portal.properties.EXAMPLE

```diff
@@ -87,6 +87,12 @@ filter_groups_by_appname=true
 googleplus.consumer.key=
 googleplus.consumer.secret=
 
+# To enable Google Analytics on the endpoints, add the appropriate Tracking ID (Format UX-XXXX-X)
```

Comment:
it may not be necessary to document these properties unless we think other installers will really want to track usage of their api.    it causes confusion with the frontend google analytics config property below (which might well be in use by others) so perhaps we keep it as hidden feature?  especially until we can be sure it doesn't have performance impact.

Reply:
I was wondering about this too.  This is fine with me.